src/main/java/com/patientpal/backend/auth/controller/AuthenticationApiV1Controller.java

```diff
@@ -1,41 +1,77 @@
 package com.patientpal.backend.auth.controller;
 
-import com.patientpal.backend.auth.dto.RefreshTokenRequest;
+import com.patientpal.backend.auth.dto.AccessTokenResponse;
 import com.patientpal.backend.auth.dto.SignInRequest;
 import com.patientpal.backend.auth.dto.SignUpRequest;
-import com.patientpal.backend.auth.dto.TokenResponse;
+import com.patientpal.backend.auth.dto.TokenDto;
 import com.patientpal.backend.auth.service.JwtLoginService;
 import com.patientpal.backend.auth.service.JwtTokenService;
 import com.patientpal.backend.member.service.MemberService;
+import com.patientpal.backend.security.jwt.JwtTokenProvider;
+import io.swagger.v3.oas.annotations.Operation;
+import io.swagger.v3.oas.annotations.headers.Header;
+import io.swagger.v3.oas.annotations.media.Content;
+import io.swagger.v3.oas.annotations.media.Schema;
+import io.swagger.v3.oas.annotations.responses.ApiResponse;
+import io.swagger.v3.oas.annotations.tags.Tag;
+import jakarta.servlet.http.Cookie;
+import jakarta.servlet.http.HttpServletResponse;
 import jakarta.validation.Valid;
 import java.net.URI;
 import lombok.RequiredArgsConstructor;
 import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.CookieValue;
 import org.springframework.web.bind.annotation.PostMapping;
 import org.springframework.web.bind.annotation.RequestBody;
 import org.springframework.web.bind.annotation.RequestMapping;
 import org.springframework.web.bind.annotation.RestController;
 
+@Tag(name = "인증", description = "인증 API")
 @RestController
 @RequestMapping("/api/v1/auth")
 @RequiredArgsConstructor
 public class AuthenticationApiV1Controller {
     private final JwtLoginService loginService;
     private final JwtTokenService tokenService;
+    private final JwtTokenProvider jwtTokenProvider;
     private final MemberService memberService;
 
     @PostMapping("/login")
-    public ResponseEntity<TokenResponse> authorize(@Valid @RequestBody SignInRequest request) {
-        return ResponseEntity.ok().body(loginService.authenticateUser(request));
+    @Operation(summary = "로그인", description = "로그인을 한다.")
+    @ApiResponse(responseCode = "200", description = "로그인 성공",
+        content = @Content(schema = @Schema(implementation = AccessTokenResponse.class)),
```

Comment:
⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.indentation.IndentationCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'content' has incorrect indentation level 8, expected level should be 12.